pyproject.toml

```diff
@@ -81,9 +81,18 @@ validators = [
     "pandas"
 ]
 
+[tool.isort]
+profile = "black"
+
```

Comment:
Ensure that isort is always run with `profile=black` even when ruining isort directly.